### ISSUE TITLE
Add WP8.x support

### ISSUE THREAD

**Issue by obenjiro** [twbs/ratchet#563]
Add support for WP8.x plz..

**Comment by jaminellis** [comment]
+1

**Comment by formikaio** [comment]
+1

**Comment by thewax** [comment]
Hello. I've modified ratchet to added WP8 touch handlers. I'd also have to add IE9/10 polyfill for CustomEvent(), otherwise push wouldn't work. I'd have to clean out some other customizations I've made, but i can submit it for review if people are still interested. I'd consider it "alpha" at best, as I have not tested this on all phones/environments. Just on a crappy 8.1 phone I have. But the code changes are pretty straightforward.

**Comment by softexe** [comment]
Ratchet is died..  Dev migrated to ionic/framework7/etc

**Comment by thewax** [comment]
Fair enough. Figured I'd post since this was one of the bigger things people were concerned about.
